src/main/java/com/opams/arch/entities/OriginEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.opams.arch.entities;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
What is the purpose of tostring, noargs and args constructor?

Reply:
to string is only for logs constructors are for creating an instance

Reply:
I mean that for current state of development we don't need noargs, args and tostring annotations